website/docs/cdktf/cli-reference/commands.mdx

```diff
@@ -1,11 +1,11 @@
 ---
-page_title: CLI Commands - CDK for Terraform
+page_title: CLI commands reference
```

Comment:
We can omit the "CDKTF" from the H1, but I think it's good to have it in the metadata!
```suggestion
page_title: CLI commands reference for CDKTF
```